motioneye/DOCS.md

```
@@ -196,16 +196,16 @@ SOFTWARE.
 
 [addon-badge]: https://my.home-assistant.io/badges/supervisor_addon.svg
 [addon]: https://my.home-assistant.io/redirect/supervisor_addon/?addon=a0d7b954_motioneye
-[contributors]: https://github.com/hassio-addons/addon-motioneye/graphs/contributors
+[contributors]: https://github.com/hassio-addons/app-motioneye/graphs/contributors
 [discord-ha]: https://discord.gg/c5DvZ4e
 [discord]: https://discord.me/hassioaddons
 [dockerhub]: https://hub.docker.com/r/hassioaddons/motioneye
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "dockerhub"

(MD053, link-image-reference-definitions)